mst/mst.go

```diff
@@ -39,7 +40,6 @@
 // If the first leaf in a tree is `bsky/posts/abcdefg` and the second is `bsky/posts/abcdehi`
 // Then the first will be described as `prefix: 0, key: 'bsky/posts/abcdefg'`,
 // and the second will be described as `prefix: 16, key: 'hi'.`
-
```

Comment:
(So the comment above is the Go package doc... needs to be attached to the package line)

Reply:
hi please i need your help my phone number is +2348069056081